js/isomer.js

```diff
@@ -72,20 +72,24 @@ Isomer.prototype._translatePoint = function(point) {
  * This method also accepts arrays
  */
 Isomer.prototype.add = function(item, baseColor) {
+  var canvasPaths = [];
+
   if (Object.prototype.toString.call(item) == '[object Array]') {
     for (var i = 0; i < item.length; i++) {
-      this.add(item[i], baseColor);
+      canvasPaths.concat(this.add(item[i], baseColor));
```

Comment:
I just realized, I used concat incorrectly here. Concat returns a new array... This should be re-assigned. I'll get to that later tonight. :smile_cat: